packages/web-components/docs/contributing-to-web-components.md

```diff
@@ -0,0 +1,150 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Table of Contents
+
+- [Contributing to Carbon for IBM.com Web Components package](#contributing-to-carbon-for-ibmcom-web-components-package)
+  - [Overview](#overview)
+  - [Packages](#packages)
+  - [JSDoc](#jsdoc)
+  - [Stable Selectors](#stable-selectors)
+  - [Feature Flag](#feature-flag)
+  - [Environment Variables](#environment-variables)
+  - [Storybook](#storybook)
+  - [Unit Test Coverage](#unit-test-coverage)
+  - [Submission Guidelines](#submission-guidelines)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Contributing to Carbon for IBM.com Web Components package
+
+> Guidelines for how to make a code contribution to the Web Components package
+
+## Overview
+
+Thank you for your contribution to Carbon for IBM.com! Below are guidelines on
+what some of the things we would be looking for as part of your contribution.
```

Comment:
We should also mention more detailed coding conventions: https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/docs/coding-conventions.md

Also should we merge [our developer doc](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.12.0/packages/web-components/docs/developer.md) to this, or add a link from here to there?

Reply:
We should link out, I think the developer doc is pretty dense so probably better to have it separate.

Reply:
Sorry, I mistaken your meaning with coding conventions. Let me see if merging these makes sense...